site/content/contribute/mobile/build-your-own/android.md

```diff
@@ -72,25 +72,25 @@ To make it easier to customize your build, we've defined a few environment varia
-| `BUILD\_FOR\_RELEASE` | Defines if the app should be built in release mode. <br><br>Valid values are: `true`, `false` <br><br>**Make sure you set this value to true if you plan to submit this app Google Play or distribute it in any other way**. | `false` | Yes |
-| `SEPARATE\_APKS` | Build one APK per achitecture (armeabi-v7a, x86, arm64-v8a and x86_64) as well as a universal APK. The advantage is the size of the APK is reduced by about 4MB. <br><br>People will download the correct APK from the Play Store based on the CPU architecture of their device. | `false` | Yes |
-| `SUBMIT\_ANDROID\_TO\_<br>GOOGLE\_PLAY` | Should the app be submitted to the Play Store once it finishes building, use along with `SUPPLY\_TRACK`.<br><br>Valid values are: `true`, `false` | `false` | Yes |
-| `SUPPLY\_TRACK` | The track of the application to use when submitting the app to Google Play Store. Valid values are: `alpha`, `beta`, `production` <br><br>**RIt is not recommended to submit the app to production. First try any of the other tracks and then promote your app using the Google Play console**. | `alpha` | Yes |
-| `SUPPLY\_PACKAGE\_NAME` | The package Id of your application, make sure it matches `MAIN\_APP\_IDENTIFIER`. |  | Yes |
-| `SUPPLY\_JSON\_KEY` | The path to the service account `json` file used to authenticate with Google.<br><br>See the [Supply documentation]( https://docs.fastlane.tools/actions/supply/#setup) to learn more. |  | Yes |
+| `COMMIT_CHANGES_TO_GIT` | Should the Fastlane script ensure that there are no changes to Git before building the app and that every change made during the build is committed back to Git. <br><br>Valid values are: `true`, `false` | `false` | No |
+| `BRANCH_TO_BUILD` | Defines the Git branch that is going to be used for generating the build. <br><br>**Make sure that, if this value is set, the branch it is set to exists**.| `$GIT_BRANCH` | No |
+| `GIT_LOCAL_BRANCH` | Defines the local branch to be created from `BRANCH_TO_BUILD` to ensure the base branch does not get any new commits on it. <br><br>**Make sure a branch with this name does not yet exist in your local Git repository**. | build | No |
+| `RESET_GIT_BRANCH` | Defines if, once the build is done, the branch should be reset to the initial state before building and whether to also delete the branch created to build the app. <br><br>Valid values are: `true`, `false` | `false` | No |
+| `VERSION_NUMBER `| Set the version of the app at build time to a specific value, rather than using the one set in the project. |  | No |
+| `INCREMENT_VERSION_<br>NUMBER_MESSAGE` | Set the commit message when changing the app version number. | Bump app version number to | No |
+| `INCREMENT_BUILD_NUMBER` | Defines if the app build number should be incremented. <br><br>Valid values are: `true`, `false` | `false` | No |
+| `BUILD_NUMBER` | Set the build number of the app at build time to a specific value, rather than incrementing the last build number. |  | No |
```

Comment:
```suggestion
| `BUILD_NUMBER` | Sets the build number of the app at build time to a specific value, rather than incrementing the last build number. |  | No |
```